programs/ssl/ssl_server2.c

```diff
@@ -1881,6 +1896,19 @@ int main( int argc, char *argv[] )
         else if( strcmp( p, "sig_algs" ) == 0 )
             opt.sig_algs = q;
 #endif
+#if defined(MBEDTLS_SSL_EARLY_DATA)
+        else if( strcmp( p, "max_early_data_size" ) == 0 )
+        {
+            long long value = atoll( q );
+            tls13_early_data_enabled =
```

Comment:
A common question, in server side, it seems we check the `max_early_data_size `to decide whether we enable early data or not, is that what we need?